README.md

```diff
@@ -1 +1,9 @@
-# live_cell_timelapse_apoptosis_analysis
\ No newline at end of file
+# Live Cell Timelapse Apoptosis Analysis
+
+This repository contains the code to analyze live cell timelapse microscopy data of apoptosis in HeLa cells.
```

Comment:
Does it make sense to include that you will be analyzing the CellProfiler morphology features and using the CellProfiler segmentation data for analyzing the images with scDINO?

I would recommend being a bit more specific here, given that you have already started scDINO analysis.

Reply:
I added more about CP and the scDINO approaches